website/www/site/layouts/shortcodes/flink_java_pipeline_options.html

```diff
@@ -77,11 +77,6 @@
   <td>Remove unneeded deep copy between operators. See https://issues.apache.org/jira/browse/BEAM-11146</td>
   <td>Default: <code>false</code></td>
 </tr>
-<tr>
-  <td><code>filesToStage</code></td>
-  <td>Jar-Files to send to all workers and put on the classpath. The default value is all files from the classpath.</td>
-  <td></td>
-</tr>
```

Comment:
Can you revert this change (and the equivalent Python one). I think something automated is trying to make this change, but we should do it in a separate PR